### PR TITLE
[php8-compact] Fix CRM_Core_FormTest testopeningforms for in php8

### PR DESCRIPTION
**Pull request by seamuslee001**
Overview
----------------------------------------
This fixes the test CRM_Core_FormTes::testopeningforms in php8

Before
----------------------------------------
Test fails in php8

After
----------------------------------------
Test passes in php8

ping @demeritcowboy @colemanw @totten

### CODE REVIEW

**Comment by seamuslee001** [comment]
its probably easier to use https://github.com/civicrm/civicrm-core/pull/20582/files?w=1 to review as that ignores the whitespace changes

File: templates/CRM/Tag/Form/Edit.tpl [hunk 1]
```diff
@@ -58,14 +58,14 @@
         {/if}
     </table>
     {else}
-        <div class="status">{ts 1=$delName}Are you sure you want to delete <b>%1</b>?{/ts}<br />{ts}This tag will be removed from any currently tagged contacts, and users will no longer be able to assign contacts to this tag.{/ts}</div>
+        <div class="status">{if !empty($delName)}{ts 1=$delName}Are you sure you want to delete <b>%1</b>?{/ts}<br />{ts}This tag will be removed from any currently tagged contacts, and users will no longer be able to assign contacts to this tag.{/ts}{/if}</div>
```
Comment (demeritcowboy) [inline templates/CRM/Tag/Form/Edit.tpl:61]: I've only quickly skimmed this PR and ignored the price ones but this one jumps out - seems like this _should_ be a fail if it's empty. In fact if it's empty it should wipe your database and tell you to start again (grin).

**Comment by seamuslee001** [comment]
I have put up an alternate at #20583 which I think might be better

**Comment by seamuslee001** [comment]
closing in favour of #20603

**Pull request closed without merging**